packages/client/src/pages/Forum/TopicConversationPage/ui/components/CommentList/components/ReactionBlock/components/ReactionOptionList/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+import { TReactionOptionList } from './types';
+import css from './styles.module.scss';
+
+export const ReactionOptionList: FC<TReactionOptionList> = props => {
+  const { schema, basicReactionData, addReaction } = props;
+
+  const handleClick = (reactionName: string) => {
+    const data = { ...basicReactionData, reactionName };
+    addReaction(data);
```

Comment:
[Предложение]
Можно без дополнительной переменной
```suggestion
    addReaction({ ...basicReactionData, reactionName });
```

Reply:
Да. 

Объясни пожалуйста, почему нельзя объявить переменную. Будет утечка памяти или работать будет медленнее из-за минимального времени операции JS? 

Я хочу показать , что  `basicReactionData` берем из замыкания

Reply:
Утечек никаких не будет, просадок по перфомансу тоже. 
Это просто подход, что не нужно создавать лишние переменные, если только они не улучшают читаемость. 

В варианте без переменной читаемость не хуже, потому что и так понятно, что происходит, строчка короткая, все удобно.
И есть даже правило для линтера, которое самостоятельно убирает переменную, если она используется один раз

Reply:
Окей